roofit/hs3/test/CMakeLists.txt

```
@@ -0,0 +1,5 @@
+ROOT_ADD_PYUNITTEST(roofit-hs3-histfactory-json histfactory.py
+  COPY_TO_BUILDDIR ${CMAKE_CURRENT_SOURCE_DIR}/histfactory-input.root
+  )
+
+ROOT_ADD_GTEST(argus argus.cxx LIBRARIES RooFitCore RooFit RooFitHS3)
```

```suggestion
ROOT_ADD_PYUNITTEST(roofit-hs3-histfactory-json test_hs3_histfactory_json.py
  COPY_TO_BUILDDIR ${CMAKE_CURRENT_SOURCE_DIR}/test_hs3_histfactory_json_input.root
  )

ROOT_ADD_GTEST(testRooFitHS3 testRooFitHS3.cxx LIBRARIES RooFitCore RooFit RooFitHS3)
```

Please change the names of the test files and unit tests to these conventional names (some file renaming that you need to do will follow from this change).